cranelift/filetests/filetests/isa/x86/simd-comparison-run.clif

```diff
@@ -1,6 +1,5 @@
 test run
 set enable_simd
-target x86_64 skylake
```

Comment:
Why is this line being removed?

Reply:
As we discussed offline, this is a `run` test, so it naturally uses the host as the target.